src/components/CourseName.tsx

```diff
@@ -0,0 +1,12 @@
+interface CourseNameProps {
+  name: string;
+}
+
+const CourseName = ({ name }: CourseNameProps) => {
+  return (
+    <div className="flex w-full flex-col items-center rounded-2xl bg-dark-cyan text-center font-Malayalam font-normal">
```

Comment:
```suggestion
    <div className="flex w-full flex-col items-center rounded-2xl bg-dark-cyan text-center font-Malayalam">
```
not needed, does nothing